packages/amplify-cli-overrides-helper/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "amplify-cli-overrides-helper",
```

Comment:
shoudl we namespace it @aws-amplify/amplify-cli-overrides-helper"?

Reply:
Yeah, you're right. Name-spaced it - @aws-amplify/cli-overrides-helper